control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -2051,6 +2071,88 @@ func (r *reconciler) ensureGuestAdmissionWebhooksAreValid(ctx context.Context) e
 	return errors.NewAggregate(errs)
 }
 
+func (r *reconciler) reconcileKubeletConfig(ctx context.Context) error {
```

Comment:
Can we // document what and why this function is doing?
Can we also makesure that any object mirrored is protected by our VAP, see ReconcileKASValidatingAdmissionPolicies